### DISCUSSION TITLE
Prevent Auth Bearer token Showing up in logs

### DISCUSSION THREAD

**Discussion by bimal10** [Netflix/conductor#3007] [General]
Currently only way to pass Auth Bearer token is in inputParameters -> headers -> authorization. The problem with this is all inputParameters get printed in logs or is exposed , is there anyway avoid this issue and prevent showing auth token

**Comment by v1r3n** [comment]
Hi @bimal10 logs --> you can prob. reduce the log level or maybe send a PR to remove headers from being printed in the logs.
Currently its not possible to hide the auth token in the UI, but we are working on a solution on that.  Will keep you posted.

**Comment by aravindanr** [comment]
@bimal10 i presume you are referring to the HTTP task. I couldn't find a log statement that prints the input data. can you share an example? Note that the inputData will be shown in the UI, there's currently no way to hide it.
